### PR TITLE
my landing page design

### PR DESCRIPTION
**Pull request by Eudoh940**
my home page design

### CODE REVIEW

File: submissions/EmmanuelUdoh/HTML/landing.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
+body{
+    box-sizing: border-box;
+}
+#wrapper{font-family: Verdana, Geneva, Tahoma, sans-serif;}
+
+.container{width: 90%;
```
Comment (kenny-wealth) [inline submissions/EmmanuelUdoh/HTML/landing.css]: indentation 🙄

File: submissions/EmmanuelUdoh/HTML/landing.css [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+     box-shadow: 10px 14px 50px 2px grey;  
+ }
+
+ nav{
+     padding: 8px;
+     width: 1280px;
+     margin: 0 auto; 
+ }
+
+ nav a{
```
Comment (kenny-wealth) [inline submissions/EmmanuelUdoh/HTML/landing.css]: Please remove spaces before the CSS declaration

File: submissions/EmmanuelUdoh/HTML/landing.css [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+     height: 40px;
+     border: none; background: orangered;
+     border-radius: 2px;
+ }
+ 
+ .sub_p{
+     color: black;
+     font-style: italic;
+     font-size: 17px;
+
```
Comment (kenny-wealth) [inline submissions/EmmanuelUdoh/HTML/landing.css]: Please remove extra space

File: submissions/EmmanuelUdoh/HTML/landingPage.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                            <li><a href="#">Google+</a></li>
+                    
+                        </ul>
+                    </center>
+                </div>
+                <div class="footer_sub_3">
+                    <center>
+                        <h2>Subscribe</h2>
+                        <input type="text" name="subs" placeholder="write your e-mail" class="subs">
+                             <input type="submit" name="submit_btn" value="subscribe" class="sub_btn">
```
Comment (kenny-wealth) [inline submissions/EmmanuelUdoh/HTML/landingPage.html]: Indentation is off here

File: submissions/EmmanuelUdoh/HTML/landingPage.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                <a href="#">Computers and Accessories</a>
+                <a href="#">Fashion</a>
+                <a href="#">Kids and Toys</a>
+               
+            </nav>
+        </div>
+        <!--home banner-->
+        <div id="banner">
+            <ul class="banner">
+                <li><img src="images/b6.jpg"></li>
```
Comment (kenny-wealth) [inline submissions/EmmanuelUdoh/HTML/landingPage.html]: Why is this a list item? It shows a huge dot right next to the banner in preview

File: submissions/EmmanuelUdoh/HTML/landing.css (hunk 1 above)
Comment (kenny-wealth) [reply submissions/EmmanuelUdoh/HTML/landing.css]: Generally review the CSS section on the style guide and make necessary corrections

**Review by kenny-wealth — CHANGES_REQUESTED** [review]

**Review by Gbensky — DISMISSED** [review]

File: submissions/EmmanuelUdoh/HTML/landing.css [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+    height: 32px;
+    background: rgb(196, 45, 196);
+    color: yellow;
+    font-family: cursive;
+    margin-top: -14px;
+} 
+#subheader p {
+    float: left;
+    margin-top: 7px;
+
```
Comment (adaugochi) [inline submissions/EmmanuelUdoh/HTML/landing.css]: please remove whitespace

File: submissions/EmmanuelUdoh/HTML/landing.css [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+    float: left;
+    margin-top: -15px;   
+}
+#logo {
+    width: 300px;
+    margin-top: 10px;
+    float: left;
+    margin-left: 200px
+}
+#ist {
```
Comment (adaugochi) [inline submissions/EmmanuelUdoh/HTML/landing.css]: This is not a good `id` name. Please use a more descriptive name

File: submissions/EmmanuelUdoh/HTML/landingPage.html (hunk 5 above)
Comment (Eudoh940) [reply submissions/EmmanuelUdoh/HTML/landingPage.html]: i have fixed this

**Review by adaugochi — APPROVED** [review]

**Review by kenny-wealth — APPROVED** [review]

**Pull request merged**